master/internal/experiment.go

```diff
@@ -113,6 +113,7 @@ type (
 		searcher            *searcher.Searcher
 		warmStartCheckpoint *model.Checkpoint
 		continueFromTrialID *int
+		continueTrials      *bool
```

Comment:
does this need to be a ```*bool``` or can it just be a ```bool```?

Reply:
I think it is better if it is`*bool` because in the case when we are creating a normal experiment, it needs to be not true. If it is a pointer we can get away with it being nil instead of setting it explicitly false. What do you think?

Reply:
done